types/cleeng.d.ts

```diff
@@ -2,7 +2,9 @@ interface ApiResponse {
   errors: string[];
 }
 type ServiceResponse<R> = { responseData: R } & ApiResponse;
+type ServiceRequest<P, R> = (payload: P) => Promise<ServiceResponse<R>>;
```

Comment:
I think these should all be Cleeng... names, right?  There are duplicates between this and account.d.ts and some are unused.

Reply:
OK, then all these changes to support setting metadata through the service.updateCustomer should be rolled back.

This means the only 'required' parameter is fullname, right? Anything else will be part of the metadata.

Reply:
I guess you meant to write this above on top of the other comment, but the metadata saving should remain in the update account because when we update full name we need to update the metadata for `first_name` and `surname` since this is how the metadata looks if you split the inplayer register fields from InPlayer dashboard.